changelog/632.doc.rst

```diff
@@ -0,0 +1 @@
+Clarify :attr:`Interaction.channel` return type.
```

Comment:
How is this clarified? This is should be more specific so a user doesn't need to hunt the diff to see what exactly changed.

Reply:
Fair enough, added more details in 2dbf94f5335bfeba4addb3b6d9aeae02730f9ece.